### PR TITLE
Add Bershka 

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: declarations/Bershka.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Bershka",
+  "documents": {
+    "Trackers Policy": {
+      "fetch": "https://static.bershka.net/4/static/itxwebstandard/docs/cookiespolicy/cookies_policy_fr_FR.pdf?ts=20220221020033"
```
Comment (martinratinaud) [inline declarations/Bershka.json:5]: why `?ts=` ?

Seems to be working without

File: declarations/Bershka.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    },
+    "Commercial Terms": {
+      "fetch": "https://static.bershka.net/4/static/itxwebstandard/docs/termsandconditions/terms_and_conditions_fr_FR.pdf?"
```
Comment (martinratinaud) [inline declarations/Bershka.json:8]: no need for ending `?`

**Review by martinratinaud — CHANGES_REQUESTED** [review]

File: declarations/Bershka.json (hunk 1 above)
Comment (clementbiron) [reply declarations/Bershka.json]: well-hunted !

File: declarations/Bershka.json (hunk 2 above)
Comment (clementbiron) [reply declarations/Bershka.json]: thanks

**Pull request merged**